src/Illuminate/Database/Query/Builder.php

```diff
@@ -227,15 +227,7 @@ public function select($columns = ['*'])
     {
         $this->columns = [];
         $this->bindings['select'] = [];
-        $columns = is_array($columns) ? $columns : func_get_args();
```

Comment:
So is it won't be possible to call it like `->select('a', 'b')` (without using an array) anymore, right? Or am I missing something?